samples/snippets/src/main/java/pubsublite/CreateSubscriptionExample.java

```diff
@@ -41,24 +42,39 @@ public static void main(String... args) throws Exception {
     String topicId = "your-topic-id";
     String subscriptionId = "your-subscription-id";
     long projectNumber = Long.parseLong("123456789");
+    boolean regional = false;
 
-    createSubscriptionExample(cloudRegion, zoneId, projectNumber, topicId, subscriptionId);
+    createSubscriptionExample(
+        cloudRegion, zoneId, projectNumber, topicId, subscriptionId, regional);
   }
 
   public static void createSubscriptionExample(
-      String cloudRegion, char zoneId, long projectNumber, String topicId, String subscriptionId)
+      String cloudRegion,
+      char zoneId,
+      long projectNumber,
+      String topicId,
+      String subscriptionId,
+      boolean regional)
       throws Exception {
 
+    CloudRegionOrZone location = null;
```

Comment:
nit: this would maybe be better as a ternary if you can fit it on one line so you can avoid it being "null".

Reply:
Ack.